dev/release/post-08-docs.sh

```diff
@@ -103,4 +103,5 @@ if [ ${PUSH} -gt 0 ]; then
   echo "Success!"
   echo "Create a pull request:"
   echo "  ${github_url}/pull/new/${branch_name}"
+  echo "Note! Use the 'asf-site' base branch for the PR!"
```

Comment:
We may be able to use `${github_url}/pull/new/asf-site...${branch_name}` for this.